### ISSUE TITLE
Limit new path commands to the currently allowed ones

### ISSUE THREAD

**Issue by MewPurPur** [MewPurPur/GodSVG#97]
- M must always be the first command (maybe it should even be added by default when making a new path)
- S should only go only after C
- T should only go only after Q
- M and Z shouldn't go after M or Z.

Commands that aren't allowed should be grayed out rather than removed.

**Comment by MewPurPur** [comment]
Already done in a WIP pull request that will take a while longer to finish.
